### DISCUSSION TITLE
Direct print output html value

### DISCUSSION THREAD

**Discussion by iqbal1410** [dolanmiu/docx#1732] [Q&A] [answered]
Hi,

Is it possible to direct style the output of the content of the value of html code? example : <span style="bold: true">Name:</span> that i get the value from database, and print the respected style when generate the docx file?

**Comment by lignar-fazmin** [comment]
hi,

i got same issue, anyone help?

**Reply by dolanmiu** [reply]
Maybe try this?

https://www.npmjs.com/package/html-to-docx

**Comment by dolanmiu** [comment, marked as answer]
Don't fully understand, but I think you are asking if `docx` can create HTML?

Sorry, this is not possible. HTML and CSS are not supported

**Reply by iqbal1410** [reply]
No, what i mean is can the docx print directly the value of the html code like `<b> this is bold</b>` will print bold with the style when generate the report.

**Reply by dolanmiu** [reply]
This sounds like html -> docx? Not possible with this library

Maybe try this?

https://www.npmjs.com/package/html-to-docx

**Reply by iqbal1410** [reply]
alright. thank you for your response and advice.
